test/unit_inbroker_parallel_SUITE.erl

```diff
@@ -574,6 +575,42 @@ login_of_passwordless_user1(_Config) ->
     passed.
 
 
+set_tags_for_passwordless_user(Config) ->
+    passed = rabbit_ct_broker_helpers:rpc(Config, 0,
+      ?MODULE, set_tags_for_passwordless_user1, [Config]).
+
+set_tags_for_passwordless_user1(_Config) ->
+    Username = <<"set_tags_for_passwordless_user">>,
+    Password = <<"set_tags_for_passwordless_user">>,
```

Comment:
See [this comment](https://github.com/rabbitmq/rabbitmq-server/pull/1468#pullrequestreview-87941249)